src/hotspot/cpu/x86/c1_Runtime1_x86.cpp

```diff
@@ -632,6 +638,14 @@ void Runtime1::initialize_pd() {
   // nothing to do
 }
 
+uint Runtime1::runtime_blob_current_thread_offset(frame f) {
+#ifdef _LP64
+  return r15_off / 2;
```

Comment:
r15_off is a byte offset, so this returns a 16-bit short offset?  I think we need a comment here to explain the / 2 and what this returns.

Reply:
I copied the same comments the other platforms have to make it more clear.